plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -0,0 +1,818 @@
+            of AP param can be changed like change all led_brightness param to 8 in config
+            eg : family: "Unified AP"
+        type: str
+        required: False
+
+      below list of AP Config param can be changes based on the requirement
+    ap_name:
+        description: AP Name changes, we need to provide the current name
+            of AP need to give in this field also ap_name_new also need to give.
+            unless the AP name never change.
+            eg : ap_name: "Test2"
+                 ap_name_new: "NFW-AP1-9130AXE"
+        type: str
+        required: False
+    ap_name_new:
+        description: AP Name name changes new name need to be added in this field.
+            we need to provide along with AP Name. so that able to change the AP Name
+            eg : ap_name: "Test2"
+                 ap_name_new: "NFW-AP1-9130AXE"
+        type: str
+        required: False
+    led_brightness_level:
+        description: AP LED brightness level field also able to modify by update
+            this field. Brightness level from 1 to 10.
+            eg : led_brightness_level: 3
+        type: int
+        required: False
+    led_status:
+        description: AP LED light need to enable or disable based on this param update.
+            this will accept only 2 state "Enabled" or "Disabled"
+            eg : led_status: "Enabled"
+        type: str
+        required: False
+    location:
+        description: Changing the location name of the AP, need to provide the data
+            in case changes required.
+            eg: location: "Bangalore"
+        type: str
+        required: False
+
+requirements:
+- dnacentersdk >= 2.4.5
+- python >= 3.10
+notes:
+  - dnacsdk Method used 
+"""
+
+EXAMPLES = r"""
+- name: Create/Update/Delete Wireless Accesspoint Configuration
+  hosts: dnac_servers
+  connection: local
+  gather_facts: no
+  vars_files:
+    - "credentials.yml"
+  tasks:
+    - name: Updating Accesspoint Configureation
+      cisco.dnac.accesspoint_workflow_manager:
+        dnac_host: "{{dnac_host}}"
+        dnac_username: "{{dnac_username}}"
+        dnac_password: "{{dnac_password}}"
+        dnac_verify: "{{dnac_verify}}"
+        dnac_port: "{{dnac_port}}"
+        dnac_version: "{{dnac_version}}"
+        dnac_debug: "{{dnac_debug}}"
+        dnac_log: True
+        dnac_log_level: DEBUG
+        config_verify: True
+        state: merged
+        config:
+          - device_fields: "id,hostname,family,type,mac_address,management_ip_address,ap_ethernet_mac_address,last_updated,up_time"
+            ap_selected_field: "mac_address,eth_mac,ap_name,led_brightness_level,led_status,location"
+            mac_address: "90:e9:5e:03:f3:40"
+            led_brightness_level: 2
+            led_status: "Enabled"
+            location: "LTTS-Bangalore"
+            ap_name: "LTTS-Test1"
+            ap_name_new: "NFW-AP1-9130AXE"
+      register: output_list
+    - name: iterate through module output (a list)
```

Comment:
Is it needed ? 
And if yes then it should also be part of config only.

Reply:
removed.